v1/users/migrations/0001_initial.py

```diff
@@ -38,7 +38,7 @@ class Migration(migrations.Migration):
                 ('display_name', models.CharField(max_length=250)),
                 ('github_username', models.CharField(blank=True, max_length=250)),
                 ('profile_image', models.URLField(blank=True, max_length=500)),
-                ('slack_username', models.CharField(blank=True, max_length=250)),
```

Comment:
We don't modify migration files. You're supposed to make new migrations

Reply:
i see, i'm not that familiar with the python django. I tried running the migrations can u check if its alright?